ITSchoolBookApp.py

```diff
@@ -1,22 +1,22 @@
 def add_book():
-    book_title = input("Add book title --> ").upper()
-
-    import csv  # importing csv library
-    with open("BooksDB.csv", mode="r") as file:
-        list_of_books = csv.DictReader(file, fieldnames=["BookTitle", "BookAuthor", "SharedWith", "IsRead", "Review"])
-        book_not_present = True
-        for book in list_of_books:
-            if book.get("BookTitle") == book_title:
-                book_not_present = False
-        if book_not_present:
-            book_author = input("Add book author --> ").upper()
-            with open("BooksDB.csv", mode="a") as file:
-                list_of_books = csv.DictReader(file, fieldnames=["BookTitle", "BookAuthor", "SharedWith", "IsRead", "Review"])
-                writer = csv.DictWriter(file, fieldnames=["BookTitle", "BookAuthor", "SharedWith", "IsRead", "Review"])
-                writer.writerow({"BookTitle": book_title, "BookAuthor": book_author, "SharedWith": "", "IsRead": "Not read", "Review": "No review added"})
-            print(f"Book '{book_title}' by {book_author} was added successfully!")
-        else:
-            print("Book already added")
+    book_title = input("Add book title (Type 'R' to return to main menu) --> ").upper()
+    if book_title != "R":
+        import csv  # importing csv library
+        with open("BooksDB.csv", mode="r") as file:
```

Comment:
I would try somehow to check if the open of the file was successful and treat if there are some exceptions

Reply:
I hope I will.. eventually